plugins/balancedLevelProvider.go

```diff
@@ -68,8 +74,14 @@ func makeBalancedLevelProvider(
 	validateSpread(carryoverInclusionProbability)
 
 	randGen := rand.New(rand.NewSource(time.Now().UnixNano()))
+
+	//set starting values for the previous-run variables
```

Comment:
this (and it's usage below) is not needed since these are uninitialized based on the comment above -- it's sometimes better to be specific but the uninitialized concept is used in a few places in kelp right now.